src/Components/Employee.js

```diff
@@ -1,15 +1,54 @@
+import { useState,useEffect } from "react";
```

Comment:
Remember, it's good to format your document or make sure you've got the right spacing while you're writing your code. It makes it easier to read through and understand where the different pieces are